### PR TITLE
Upgrade Atlas go client

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ 

Closes #[issue number]

## Checklist

- [] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [] I have added tests that prove my fix is effective or that my feature works
- [] I have added any necessary documentation (if appropriate)
- [] I have run `make fmt` and formatted my code

### CODE REVIEW

File: opsmngr/opsmngr.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ import (
 )
 
 const (
-	defaultBaseURL = atlas.CloudURL
```
Comment (andreaangiolillo) [inline opsmngr/opsmngr.go:35]: `atlas.CloudURL` is not longer available

Comment (andreaangiolillo) [reply opsmngr/opsmngr.go:35]: Should I add again `atlas.CloudURL` in the atlas go client? @gssbzn I didn’t notice that it was public when I removed it

Comment (gssbzn) [reply opsmngr/opsmngr.go:35]: yes, please add it again

Comment (andreaangiolillo) [reply opsmngr/opsmngr.go:35]: https://github.com/mongodb/go-client-mongodb-atlas/pull/214

**Review by threebee — APPROVED** [review]
LGTM!

**Pull request merged**